gdcdictionary/schemas/raw_methylation_array.yaml

```diff
@@ -11,7 +11,7 @@ description: >-
   Data file containing channel data from a methylation array.
 additionalProperties: false
 submittable: true
-downloadable: false
+downloadable: true 
```

Comment:
There's a trailing space after `true`.  I'm not sure if it'll affect anything downstream, but it'll be safer to remove it.

Reply:
Good catch!  Should be gone now.